### PR TITLE
[Doc Improvement][M365 Overview][2102108]

### PR DESCRIPTION
**Pull request by v-dreddipogu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f2ba551e0d1e1cbc0bd21c5724edc86b65d595d0/msteams-platform/m365-apps/overview.md) | 94 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4684a597-3a19-441e-b6c7-877547fe38a6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3d36e06d8b5e9d7ffaddd8128316a6330fd6cd1e/msteams-platform/m365-apps/overview.md) | 94 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/59db7b4a-a156-4883-a5c0-14277e26a932) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ea9d79c4a29a1abf9a9b57bf95baa177a3d6d160/msteams-platform/m365-apps/overview.md) | 94 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4c06ce07-2062-479d-a03f-5ff3725bb463) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3b8cf08d4e566947fd336010a7f7aaf798c5b40e/msteams-platform/m365-apps/overview.md) | 93 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/983ccaa0-0766-433b-a0ba-53e787b9dd35) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/m365-apps/overview.md [hunk 1]
```diff
@@ -7,32 +7,33 @@ ms.topic: overview
 # Extend Teams apps across Microsoft 365
 
 > [!NOTE]
-> This early developer preview is intended to provide Teams developers with existing applications the chance to try the new functionality and [provide feedback](/microsoftteams/platform/feedback) on this expansion of the Teams developer platform into other high-usage areas of the Microsoft 365 ecosystem.
+> Teams developers can try the new functionalities and [provide feedback](/microsoftteams/platform/feedback) using existing applications in early Developer preview for expansion of the Teams developer platform into other high-usage areas of the Microsoft 365 ecosystem.
```
Comment (surbhigupta12) [inline msteams-platform/m365-apps/overview.md:10]: Incorrect. Changed the meaning. Please rephrase.

File: msteams-platform/m365-apps/overview.md [hunk 2]
```diff
@@ -7,32 +7,33 @@ ms.topic: overview
...
 
 :::image type="content" source="images/outlook-office-teams-personal-tab.png" alt-text="Personal tab running in outlook.com":::
 
 ## Teams messaging extensions in Outlook
 
-Also with this preview, you can extend your search-based Teams messaging extensions to Outlook on the web and Windows desktop, enabling customers to search and share results through the compose message area of Outlook, in addition to Microsoft Teams clients.
+Messaging extension search commands allow users to search external systems and share the results through compose message of Outlook on web and Windows desktop, and Teams.
 
-After sideloading to Teams, your messaging extension appears as one of your installed apps within the Outlook compose message area.
+:::image type="content" source="images/messaging-extensions-outlook-teams.png" alt-text="Messaging extensions in outlook and Teams":::
```
Comment (surbhigupta12) [inline msteams-platform/m365-apps/overview.md:31]: Please revert/rephrase. The main context is lost.

**Review by surbhigupta12 — CHANGES_REQUESTED** [review]
Please rework. The meaning is changed and context is lost in most of the places.

**Comment by v-dreddipogu** [comment]
Invalid PR. Hence closing it

**Pull request closed without merging**